src/backend/tests/conftest.py

```diff
@@ -146,7 +150,29 @@ def override_get_session() -> Generator[Session, Any, None]:
 
 @pytest.fixture
 def user(session: Session) -> User:
-    return get_factory("User", session).create(id="1")
+    return get_factory("User", session).create()
+
+
+@pytest.fixture
+def organization(session: Session) -> Organization:
+    return get_factory("Organization", session).create()
+
+
+@pytest.fixture
+def deployment(session: Session) -> Deployment:
+    return get_factory("Deployment", session).create(
+        deployment_class_name="CohereDeployment"
+    )
+
+
+@pytest.fixture
+def model(session: Session) -> Model:
+    return get_factory("Model", session).create()
+
+
+@pytest.fixture
+def agent(session: Session) -> Agent:
+    return get_factory("Deployment", session).create()
```

Comment:
This fixture should return an agent based on the signature, but we are returning a Deployment instead

Reply:
I don't think it's used anywhere, in this case it's ok to not add the fixture and delete it

Reply:
Tyvm